docs/fastdds/library_overview/includes/functionalities.rst

```diff
@@ -0,0 +1,92 @@
+the :ref:`discovery` section.
+
+Security
+^^^^^^^^
+
+Fast DDS can be configured to provide secure communications by implementing pluggable security at three levels:
+
+* Authentication of remote DomainParticipants.
+  The **DDS:Auth:PKI-DH** plugin provides authentication using a trusted Certificate
+  Authority (CA) and ECDSA Digital Signature Algorithms to perform the mutual authentication.
+  It also establishes a shared secret using Elliptic Curve Diffie-Hellman (ECDH) Key Agreement protocol.
+* Access control of entities.
+  The **DDS:Access:Permissions** plugin provides access control to DomainParticipants at the DDS Domain and Topic level.
+* Encryption of data.
+  The **DDS:Crypto:AES-GCM-GMAC** plugin provides authenticated encryption using Advanced Encryption Standard (AES) in
+  Galois Counter Mode (AES-GCM).
+
+You can continue reading the :ref:`security` section for more information.
+
+Logging
+^^^^^^^
+
+Fast DDS provides an extensible Logging system.
+:class:`Log` class is the entry point of the Logging system.
+It exposes three macro definitions to ease its usage: ``logInfo``, ``logWarning`` and ``logError``.
+Moreover, it allows the definition of new categories, in addition to those already available
+(:class:`INFO_MSG`, :class:`WARN_MSG` and :class:`ERROR_MSG`).
+It provides filtering by category using regular expressions, as well as control of the verbosity of the Logging system.
+Details of the possible Logging system configurations can be found in the :ref:`here <dds_layer_core_logging>` section.
+
+
+XML profiles configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Fast DDS offers the possibility to make changes in its default settings by using XML profile configuration files.
+Thus, the behavior of the DDS Entities can be modified without the need for the user to implement any program source
+code.
+
+The user has XML tags for each of the API functionalities.
+Therefore, it is possible to build and configure DomainParticipant profiles through the ``<participant>`` tag, or
+the DataWriter and DataReader profiles associated to the ``<data_writer>`` and ``<data_reader>`` tags respectively.
+
+For a better understanding of how to develop these XML profiles configuration files you can continue reading
+the :ref:`xml_profiles` section.
+
+Environment variables
+^^^^^^^^^^^^^^^^^^^^^
+
+Environment variables are those variables that are defined outside the scope of the program, through operating system
+functionalities.
+Fast DDS relies on two environment variables so that the user can easily customize the default settings of DDS
+applications.
+These two environment variables are as follows:
+
+* ``FASTRTPS_DEFAULT_PROFILES_FILE``.
+  Defines the location of the profile configuration XML files.
+
+* ``ROS_DISCOVERY_SERVER``.
+  Sets as the default discovery protocol the Server-Client Discovery.
+  It lists the UDP addresses of the DomainParticipant that perform the server function.
```

Comment:
This is not what is going to do any more. It's going to be just a list of locators (IP-port pairs) where servers are. The server in this case can be a user configured participant, or one created by a binary we are going to provide. In this sense, the variable configures the participants as clients to the servers in the list